.gitignore

```diff
@@ -18,7 +18,7 @@
 # The .vscode folder contains launch configuration and tasks you configure in
```

Comment:
removed the ignore for the .vscode folder and added one for .vscode/settings.json (as settings won't be shared)